CMakeLists.txt

```diff
@@ -142,49 +155,61 @@ foreach (cminpack_lib ${cminpack_libs})
 
   # Link with BLAS library if requested
   if (USE_BLAS)
-    if (${cminpack_lib} STREQUAL cminpackld)
+    if (${cminpack_lib_name} STREQUAL cminpackld)
       message (WARNING "BLAS cannot be used for the extended precision version")
     else ()
       find_package (BLAS)
       if (BLAS_FOUND)
         target_link_libraries (${cminpack_lib} PUBLIC ${BLAS_LIBRARIES})
-        set_target_properties (${cminpack_lib} PROPERTIES LINK_FLAGS "${BLAS_LINKER_FLAGS}")
         target_compile_definitions (${cminpack_lib} PUBLIC USE_BLAS)
+        list(APPEND PC_CMINPACK_CFLAGS "-DUSE_BLAS")
       endif ()
     endif ()
   endif ()
 
-  install (TARGETS ${cminpack_lib} EXPORT CMinpackConfig
-    LIBRARY DESTINATION ${CMINPACK_LIB_INSTALL_DIR} COMPONENT library
-    ARCHIVE DESTINATION ${CMINPACK_LIB_INSTALL_DIR} COMPONENT library
-    RUNTIME DESTINATION bin COMPONENT library)
   install (FILES ${cminpack_hdrs} DESTINATION ${CMINPACK_INCLUDE_INSTALL_DIR}
     COMPONENT cminpack_hdrs)
 
-  set_target_properties (${cminpack_lib} PROPERTIES
-    POSITION_INDEPENDENT_CODE ON
-    VERSION ${CMINPACK_VERSION}
-    SOVERSION ${CMINPACK_SOVERSION}
+  if (USE_BLAS AND BLAS_FOUND)
+    set_target_properties (${cminpack_lib} PROPERTIES
+      LINK_FLAGS "${BLAS_LINKER_FLAGS}"
```

Comment:
you use target_link_options instead (bump cmake version to 3.13 which is old enough)